js/src/jupyter-leaflet.css

```diff
@@ -126,3 +126,30 @@
     color: var(--jp-ui-font-color1);
     font-size: 11px;
 }
+/* Legend Control */
+div.leaflet-control-legend {
+	color: #555;
+   	line-height: 18px;
+   	width: auto;
+   	background-color: #fff;
```

Comment:
We will want to use CSS variables here for some attributes: `background-color: var(--jp-layout-color1);`. So that it integrates better with JupyterLab, and it adds automatic support for JupyterLab themes ;)

Reply:
annnnd done in a66e9cc :)